config/locales/details.yml

```diff
@@ -287,23 +297,34 @@ en:
       documents_upload_problems:
         show:
           heading: Complete your appeal by post
-          lead_text: Send everything so that it arrives within the 30 day deadline, unless you are appealing late.
+          lead_text: Send everything so that it arrives within the 30 day deadline, unless you are appealing late.      
           post_text_html:
-          
-           <p><strong>1. Print out your appeal using the following link</strong></p>
-          save_or_print_pdf: Save or print your case details
+            <div class="letter">
+              <h2 class="heading-medium">What you will need to do</h2>
+              <ol class="list list-number">
+                 <strong><li>Print out your appeal details</li></strong>
+                  <p>Use the following link to print your details:</p>
+
+                  <p><strong><a href="%{download_pdf_url}" target="_blank">Print your appeal details</a></strong></p>
+
+                  <strong><li>Make copies of all documents you wanted to upload</li></strong>
+                  <p>You must include:</p>
+                  <ul class="list list-bullet">
+                  <li>the original notice or review conclusion letter</li>
+                  <li>copies of any documents you attached such as the grounds of appeal, reasons for a late appeal, or authorising a representative</li>
+                  </ul>
+
+                 <strong><li>Send everything by post</li></strong>
+                   <p>Put your printed appeal details and copies of your documents in one envelope and post to:</p>
+                  <p>First-tier Tribunal (Tax Chamber)<br>
+                  PO Box 16972<br>
```

Comment:
We should probably have this postal address defined as a config setting, too, since we're doing that for the phone number